conf/provisioning/dashboards/nuodb/disk.json

```diff
@@ -3562,7 +3813,7 @@
           "measurement": "JrnlBytes",
           "orderByTime": "ASC",
           "policy": "default",
-          "query": "SELECT mean(\"rate\") FROM \"JrnlWrites\" WHERE (\"nodetype\" = 'Storage' AND \"db\" =~ /^$database$/ AND \"host\" =~ /^$host$/) AND $timeFilter GROUP BY time($__interval), \"host\" fill(0)",
+          "query": "SELECT mean(\"rate\") FROM \"JrnlWrites\" WHERE (\"nodetype\" = 'Storage' AND \"db\" =~ /^$database$/ AND \"host\" =~ /^$host$/) AND $timeFilter GROUP BY time($__interval), \"host\", \"db\", \"nodeid\", \"startid\" fill(0)",
```

Comment:
Why not `fill(null)` like the other queries?

Reply:
should probably be.  although there are times when using stack graphs that fill(null) has problems.   So it might have been changed to address that.

Reply:
I think the time you get in trouble with fill(null) is when using stacked bar graphs.  So we should be good here.